ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/transaction/ApiTransactionService.scala

```diff
@@ -18,37 +18,42 @@ import com.digitalasset.platform.sandbox.services.transaction.SandboxEventIdForm
 import com.digitalasset.platform.server.api.services.domain.TransactionService
 import com.digitalasset.platform.server.api.services.grpc.GrpcTransactionService
 import com.digitalasset.platform.server.api.validation.ErrorFactories
+
 import io.grpc._
-import org.slf4j.LoggerFactory
 import scalaz.syntax.tag._
-import com.digitalasset.ledger.api.v1.transaction_service.{TransactionServiceLogging}
+import com.digitalasset.ledger.api.v1.transaction_service.TransactionServiceLogging
+import com.digitalasset.platform.common.logging.NamedLogger
 
 import scala.concurrent.{ExecutionContext, Future}
 
 object ApiTransactionService {
 
-  def create(ledgerId: LedgerId, transactionsService: IndexTransactionsService)(
+  def create(
+      ledgerId: LedgerId,
+      transactionsService: IndexTransactionsService,
+      namedLogger: NamedLogger)(
       implicit ec: ExecutionContext,
       mat: Materializer,
       esf: ExecutionSequencerFactory)
     : GrpcTransactionService with BindableService with TransactionServiceLogging =
     new GrpcTransactionService(
-      new ApiTransactionService(transactionsService),
+      new ApiTransactionService(transactionsService, namedLogger = namedLogger),
       ledgerId,
       PartyNameChecker.AllowAllParties
     ) with TransactionServiceLogging
 }
 
 class ApiTransactionService private (
     transactionsService: IndexTransactionsService,
-    parallelism: Int = 4)(
+    parallelism: Int = 4,
+    namedLogger: NamedLogger)(
```

Comment:
Since this is an internal API (hence we can change it), please make `parallelism` the last parameter and remove the named parameter passing in line 40.

Reply:
Done